consensus/clique/clique.go

```diff
@@ -613,7 +613,7 @@ func (c *Clique) Prepare(ctx context.Context, chain consensus.ChainReader, heade
 	var deadline *time.Time
 	if c.config.Period != 0 {
 		t := time.Unix(header.Time.Int64(), 0)
-		t = t.Add(-time.Duration(c.config.Period) * time.Second / 2)
+		t = t.Add(-time.Duration(c.config.Period) * time.Second / 3)
```

Comment:
I think it needs to be 2/3, since we're subtracting from the end of the period here.
```suggestion
		t = t.Add(-time.Duration(c.config.Period) * time.Second * 2 / 3)
```